app/queries/my_facilities/registrations_query.rb

```diff
@@ -22,26 +22,28 @@ def initialize(facilities: Facility.all, period: :quarter, last_n: 3)
   def registrations
     @registrations ||=
       PatientRegistrationsPerDayPerFacility
-      .where(facility: @facilities)
-      .where("(year, #{@period}) IN (#{periods_as_sql_list})")
+        .where(facility: @facilities)
+        .where("(year, #{@period}) IN (#{periods_as_sql_list})")
   end
 
   def all_time_registrations
     @all_time_registrations ||=
       LatestBloodPressuresPerPatient
-      .where(registration_facility_id: @facilities)
-      .where('patient_recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
+        .where(registration_facility_id: @facilities)
+        .where('patient_recorded_at < ?', Time.current.beginning_of_day - REGISTRATION_BUFFER)
   end
 
   private
 
   def period_list(period, last_n)
```

Comment:
Can be removed now that `PeriodHelper` is a thing.

Reply:
I tried. I couldn't because the `PeriodHelper` in `RegistrationsQuery` is not inclusive for the `day`

Reply:
Could we make the helper `period_list` take `inclusive` as a key argument (defaults to true)? That could make this work.

Reply:
I could, but then I think `RegistrationsQuery` wants inclusive to be true for `month` and `quarter` but not `day`. So that parameter will not be precise.